syncer.go

```diff
@@ -65,6 +65,8 @@ func (s *Syncer) LoadClients() error {
 	if err != nil {
 		return err
 	}
+	s.logger.WithField("count", len(newConfigs)).Info("Loaded configs")
```

Comment:
What's the count field? Does this just show how many config lines were parsed? If so, can the comment reflect that?